app/scripts/app.js

```diff
@@ -42,6 +47,19 @@ angular.module('yourStlCourts').config(function($stateProvider, $urlRouterProvid
         }
       }
     })
+    .state('courtSearchInfo', {
+      url: '/courtSearch/info',
+      templateUrl: 'views/courtSearchInfo.html',
+      controller: 'courtSearchInfoCtrl as ctrl',
+      params: {
+        courtInfo: {value: undefined}
+      },
+      resolve: {
+        courtInfo : function($stateParams){
+          return $stateParams.courtInfo;
```

Comment:
This is fine for now, but we may want to consider the fact that this page can be linked via URL which we could include the court id and use that as stateParams to fetch the court info vs making the route invalid unless coming from a different route.